src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { BsFillBugFill } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import Form from '../Form/Form';
+import './Navbar.css';
+
+const Navbar = () => (
+   <>
+     <nav className="navbar navbar-expand-lg bg-body-tertiary">
+       <div className="container-fluid">
+         <a className="navbar-brand" href="/">
+           <BsFillBugFill className="logo" /> Polili
+         </a>
```

Comment:
- [ ] I would advise you to use the Link of the react-router-dom because the tag makes the page reload which makes the SPA not consistent.

Reply:
Well Noted, thank u for pointing out the things that I need to improved my code. Thanks for your time and efforts.